packages/rulesets/src/spectral/az-dataplane.ts

```diff
@@ -495,6 +503,136 @@ const ruleset: any = {
         function: hostParameters,
       },
     },
+    AvoidMsdnReferences: {
```

Comment:
if the rule is for arm & dataplane as below , you need to move it the az-common.ts, please help check all the rules
![image](https://user-images.githubusercontent.com/59603451/190355954-6df8d898-eafd-44b2-a0ce-9537defc3544.png)